Formula/cpl.rb

```diff
@@ -0,0 +1,36 @@
+class Cpl < Formula
+  desc "ISO-C libraries for developing astronomical data-reduction tasks"
+  homepage "https://www.eso.org/sci/software/cpl/index.html"
+  url "ftp://ftp.eso.org/pub/dfs/pipelines/libraries/cpl/cpl-7.1.tar.gz"
+  sha256 "ba7f762fac3550e9d9246b7fbacba54bfba4c47b833720834bd4cd21ed0e8b00"
+
+  depends_on "cfitsio"
```

Comment:
cfitsio and fftw are runtime dependencies?

Reply:
As far as I can tell, yes. `brew linkage` reports linkage to both `fftw` and `cfitsio`.

Reply:
Can we specify their locations in configure? Just to make it explicit?

Reply:
This is done now (explicitly specified locations in `configure`).